app/controllers/admin/payments_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Admin::PaymentsController < ApplicationController
+  before_filter :require_logged_in
+  before_filter :require_admin
```

Comment:
I would suggest to build an AdminController and make all other `Admin::...` controllers to inherit from that one. To have better control over `require_admin`.